lib/src/repository/traits/lock_repository.dart

```diff
@@ -1,8 +1,28 @@
+import 'package:yonomi_platform_sdk/src/queries/devices/get_device/query.data.gql.dart';
+import 'package:yonomi_platform_sdk/src/queries/devices/get_devices/query.data.gql.dart';
 import 'package:yonomi_platform_sdk/src/queries/lock/make_lock_unlock_action_request/query.req.gql.dart';
 import 'package:yonomi_platform_sdk/src/repository/base_repository.dart';
 import 'package:yonomi_platform_sdk/yonomi-sdk.dart';
 
 class LockRepository {
+  static LockTrait getLockTrait(trait) {
+    if (trait is GgetDeviceData_device_traits__asLockDeviceTrait ||
+        trait
+            is GgetDevicesData_me_devices_edges_node_traits__asLockDeviceTrait) {
+      var properties = {
```

Comment:
Didn't we decide to make classes for properties instead of maps?

Reply:
These are creating two sets and passing them to the LockTrait class. See Line 49 for this file
and constructor for `Thermostat_Repository.dart`:
[https://github.com/Yonomi/yonomi-sdk-dart/blob/175eb798c06074e8dd0fb999dafc119bcaf79946/lib/src/repository/traits/thermostat_repository.dart#L96](https://github.com/Yonomi/yonomi-sdk-dart/blob/175eb798c06074e8dd0fb999dafc119bcaf79946/lib/src/repository/traits/thermostat_repository.dart#L96)